pkg/services/folder/folderimpl/sqlstore_test.go

```diff
@@ -193,13 +193,23 @@ func TestIntegrationUpdate(t *testing.T) {
 
 	orgID := CreateOrg(t, db)
 
-	// create folder
+	// create parent folder
+	parent, err := folderStore.Create(context.Background(), folder.CreateFolderCommand{
+		Title:       folderTitle,
+		Description: folderDsc,
+		OrgID:       orgID,
+		UID:         util.GenerateShortUID(),
+	})
+
+	// create subfolder
```

Comment:
This test doesn't seem to have the `nestedFolders` feature flag enabled. I'm surprised there are no issues with creating the subfolder. Am I missing something here?

Reply:
the `folderStore` is the nested folder store (manages the `folder` table)

Reply:
Ah yes, that's right.